cmd/egress/cmd.go

```diff
@@ -42,6 +43,13 @@ func NewCmdValidateEgress() *cobra.Command {
 
 	validateEgressCmd := &cobra.Command{
 		Use: "egress",
+		Short: "Verify essential openshift domains are reachable from given subnet ID",
+		Long: dedent.Dedent(`Verify essential openshift domains are reachable from given VPC subnet ID.
+Each subnet must be verified individually.`),
```

Comment:
This is a bit confusing; each subnet of what? The whole VPC? Those provided for the cluster? Why must each be verified individually? This text is specific to the test, which is only aware of one subnet, so this message may be better in the SOP instead.

Reply:
changed to remove the SOP like line